prisma/schema.prisma

```diff
@@ -802,3 +814,98 @@ model ConsultationCommentUpvote {
   @@index([commentId])
   @@index([userId])
 }
+
+// Connects a user to a meeting (via a cityId and meetingId). Has a medium (email or message) and a type (beforeMeeting or afterMeeting).
+// Has many NotificationSubjects, which are the subjects that the notification is about.
+// Has many NotificationDeliveries, which are the planned (pending) and attempted (sent or failed) deliveries of the notification.
+model Notification {
+  id        String   @id @default(cuid())
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  user      User     @relation(fields: [userId], references: [id], onDelete: Cascade)
+  userId    String
+
+  city      City     @relation(fields: [cityId], references: [id], onDelete: Cascade)
+  cityId    String
+
+  meeting   CouncilMeeting @relation(fields: [meetingId, cityId], references: [id, cityId], onDelete: Cascade)
+  meetingId String
+
+  type NotificationType
+
+  deliveries NotificationDelivery[]
+  subjects NotificationSubject[]
+
+  @@unique([userId, cityId, meetingId, type])
+  @@index([userId])
+  @@index([cityId, meetingId, type])
+  @@index([createdAt])
+}
+
+enum NotificationType {
+  beforeMeeting
+  afterMeeting
+}
```

Comment:
Let's keep a single pattern on enums. Up until now we were using capital case, like
```suggestion
enum NotificationType {
  BEFORE_MEETING
  AFTER_MEETING
}
```

All your newly added enums follow this new lowercase pattern. I guess it might be more complex to do such a change now that changes are already on staging.